src/dvui.zig

```diff
@@ -7434,15 +7434,15 @@ pub const BasicLayout = struct {
                 if (e.isVertical()) {
                     self.seen_expanded = true;
                 }
-                r.y = self.pos;
-                r.h = @max(0, r.h - r.y);
+                r.y += self.pos;
```

Comment:
This looks a bit weird to me but don't have time to dig into it yet.  Just wanted to let you know - thanks!